installation/language/de-DE/de-DE.ini

```diff
@@ -3,123 +3,92 @@
+MIN_PHP_ERROR_TEXT="Ihr Host muss die PHP-Version {{phpversion}} oder neuer verwenden, um diese Version von Joomla auszuführen."
+MIN_PHP_ERROR_URL_TEXT="Hilf mir, dies zu lösen."
+;End of minimum PHP error page
+
+;Main Config
+INSTL_SELECT_INSTALL_LANG="Installationssprache auswählen"
 INSTL_SELECT_LANGUAGE_TITLE="Sprachauswahl"
 INSTL_WARNJAVASCRIPT="Warnung! JavaScript muss für eine reibungslose Installation von Joomla! aktiviert sein!"
 INSTL_WARNJSON="In der PHP-Umgebung muss JSON aktiviert sein, damit Joomla! installiert werden kann!"
+INSTL_SETUP_LOGIN_DATA="Zugangsdaten einrichten"
 
-;Preinstall view
-INSTL_PRECHECK_TITLE="Installationsprüfung"
-INSTL_PRECHECK_DESC="Sollte einer der Einträge vom Server nicht unterstützt werden, mit einem „<strong class=\"label label-important\">Nein</strong>“ gekennzeichnet, bitte die Einstellungen auf dem Server prüfen und anpassen.<br />Joomla! kann nicht installiert werden, wenn die unten aufgeführten Systemvoraussetzungen nicht erfüllt sind."
-INSTL_PRECHECK_RECOMMENDED_SETTINGS_TITLE="Empfohlene Einstellungen:"
-INSTL_PRECHECK_RECOMMENDED_SETTINGS_DESC="Diese Einstellungen werden für PHP empfohlen, um eine gute Kompatibilität mit Joomla! zu gewährleisten.<br />Jedoch kann Joomla! hier mit Einschränkungen in den Empfehlungen trotzdem funktionieren."
+;Precheck view
+INSTL_DATABASE_SUPPORT="Datenbankunterstützung:"
+INSTL_JSON_SUPPORT_AVAILABLE="JSON-Support"
+INSTL_MB_LANGUAGE_IS_DEFAULT="PHPs mbstring Spracheinstellung ist neutral"
+INSTL_MB_STRING_OVERLOAD_OFF="PHPs „mbstring function overload“ ist deaktiviert"
+INSTL_NOTICE_DATABASE_SUPPORT="Es wurden keine unterstützten Datenbanken gefunden."
+INSTL_NOTICE_JSON_SUPPORT_AVAILABLE="In der PHP-Umgebung muss JSON aktiviert sein, damit Joomla! installiert werden kann!"
+INSTL_NOTICE_MBLANG_NOTDEFAULT="PHPs mbstring Spracheinstellung ist nicht auf „neutral“ gesetzt. Dies kann lokal eingestellt werden, indem Sie <strong>php_value mbstring.language neutral</strong> in Ihre <code>.htaccess</code> Datei eingeben."
+INSTL_NOTICE_MBSTRING_OVERLOAD_OFF="PHPs „mbstring function overload“ ist aktiviert. Dies kann lokal deaktiviert werden, indem Sie <strong>php_value mbstring.func_overload 0</strong> in Ihre <code>.htaccess</code> Datei eingeben."
+INSTL_NOTICE_PARSE_INI_FILE_AVAILABLE="Die erforderlichen php-Funktionen <code>parse_ini_file</code> und <code>parse_ini_string</code> sind auf Ihrem Server deaktiviert."
+INSTL_NOTICE_XML_SUPPORT="Die XML-Unterstützung ist nicht verfügbar. Dies sollte standardmäßig in php aktiviert sein, Ubuntu-Benutzer müssen möglicherweise durch Ausführen von <code>sudo apt-get install php-xml</code> die benötigten Pakete installieren, gefolgt von einem Neustart des Webservers."
+INSTL_NOTICE_ZLIB_COMPRESSION_SUPPORT="Die Zlib-Kompression ist nicht eingestellt. Dies kann lokal aktiviert werden, indem Sie <strong>zlib.output_compression = On</strong> in Ihre <code>php.ini</code> Datei eingeben."
```

Comment:
INSTL_NOTICE_ZLIB_COMPRESSION_SUPPORT="Die Zlib-Kompression ist nicht eingestellt. Dies kann lokal aktiviert werden, indem <strong>zlib.output_compression = On</strong> in die <code>php.ini</code> Datei eingefügt wird."